sklearn/linear_model/tests/test_bayes.py

```diff
@@ -110,6 +111,22 @@ def test_std_bayesian_ridge_ard_with_constant_input():
         assert_array_less(y_std, expected_upper_boundary)
 
 
+def test_regression_issue_10128():
+    # this ARDRegression test throws a `ValueError` on master, commit 5963fd2
```

Comment:
Maybe a succinct description will be better. you can also put the issue number instead of the commit

Reply:
done